v4/weaviate/testenv/environment.go

```diff
@@ -42,7 +44,7 @@ func SetupLocalWeaviateDeprecated() error {
 	return WaitForWeaviate()
 }
 
-func isExternalWeaviateRunning() bool {
+func IsExternalWeaviateRunning() bool {
```

Comment:
is this function being used somewhere outside this package? if not then I would make it private